sfall/Modules/MiscPatches.cpp

```diff
@@ -855,6 +873,21 @@ void KeepWeaponSelectModePatch() {
 	}
 }
 
+void PartyMemberSkillPatch() {
+	// Fixed getting distance from source to target when using skills
+	// Note: You need to use the Doctor/First-Aid skill on the player, this will cause the party member to apply his skill,
+	// but the engine function is not implemented completely so using this fix without overriding the Doctor/First-Aid
+	// functions in the global script is impractical, so this should be disabled by default
+	if (GetConfigInt("Misc", "PartyMemberSkillFix", 0) != 0) {
+		dlog("Applying party member using skill Doctor/First-Aid patch.", DL_INIT);
+		HookCall(0x412836, action_use_skill_on_hook);
+		dlogr(" Done", DL_INIT);
+	}
+	// Small code fix (change obj_dude to source)
```

Comment:
Does this fix address any vanilla bug or just for making the hook work properly?

Reply:
For the hook, for the game itself there is no difference what to use source or obj_dude